### ISSUE TITLE
✨ [Feature request]: Create an OG image for DoC site

### ISSUE THREAD

**Issue by Sachin-chaurasiya** [reactplay/docs#49]
### Is your feature request related to a problem? Please describe.

Currently we don’t have an OG image for doc site and because of that the gist of site is not looking good.

### Describe the solution you'd like.

Creat an OG image for doc site.

### Describe alternatives you've considered.

Na

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by priyankarpal** [comment]
I want to work on it. could you assign me?

**Comment by Sachin-chaurasiya** [comment]
> I want to work on it. could you assign me?

Assigned to you @priyankarpal .

**Comment by priyankarpal** [comment]
simply we can add the reactplay doc website's screenshots. what do you think? @Sachin-chaurasiya

**Comment by Sachin-chaurasiya** [comment]
> simply we can add the reactplay doc website's screenshots. what do you think? @Sachin-chaurasiya

I think we should have something like this image.

![image](https://user-images.githubusercontent.com/59080942/229703844-abe21374-e3e1-4301-89bb-93e17bc68e64.png)

**Comment by atapas** [comment]
> > simply we can add the reactplay doc website's screenshots. what do you think? @Sachin-chaurasiya
> 
> I think we should have something like this image.
> 
> ![image](https://user-images.githubusercontent.com/59080942/229703844-abe21374-e3e1-4301-89bb-93e17bc68e64.png)

I am thinking that we should customize it for our all sister sites! Like for docs, its the same template but it should a documentation somewhere... similarly for hustles, it should say events. Makes sense?

**Comment by priyankarpal** [comment]
> > > simply we can add the reactplay doc website's screenshots. what do you think? @Sachin-chaurasiya
> > 
> > 
> > I think we should have something like this image.
> > ![image](https://user-images.githubusercontent.com/59080942/229703844-abe21374-e3e1-4301-89bb-93e17bc68e64.png)
> 
> I am thinking that we should customize it for our all sister sites! Like for docs, its the same template but it should a documentation somewhere... similarly for hustles, it should say events. Makes sense?

sounds good 🎉

**Comment by Sachin-chaurasiya** [comment]
@priyankarpal , How we are doing here? any progress?

**Comment by priyankarpal** [comment]
> > > simply we can add the reactplay doc website's screenshots. what do you think? @Sachin-chaurasiya
> > 
> > I think we should have something like this image.
> > 
> > ![image](https://user-images.githubusercontent.com/59080942/229703844-abe21374-e3e1-4301-89bb-93e17bc68e64.png)
> 
> I am thinking that we should customize it for our all sister sites! Like for docs, its the same template but it should a documentation somewhere... similarly for hustles, it should say events. Makes sense?

We can customise it

**Comment by Sachin-chaurasiya** [comment]
> > > > simply we can add the reactplay doc website's screenshots. what do you think? @Sachin-chaurasiya
> > > 
> > > 
> > > I think we should have something like this image.
> > > ![image](https://user-images.githubusercontent.com/59080942/229703844-abe21374-e3e1-4301-89bb-93e17bc68e64.png)
> > 
> > 
> > I am thinking that we should customize it for our all sister sites! Like for docs, its the same template but it should a documentation somewhere... similarly for hustles, it should say events. Makes sense?
> 
> We can customise it

Yeah, that we already discussed, can you open a PR with the changes?

**Comment by priyankarpal** [comment]
Currently my exams are going on so you can unassign me 😅

**Comment by Sachin-chaurasiya** [comment]
> Currently my exams are going on so you can unassign me 😅

Okay, no worries, however, it's good Open Source practice to keep the community updated.

**Comment by jayanth137** [comment]
Hey @Sachin-chaurasiya , Can you assign me? I can work on it.

**Comment by Sachin-chaurasiya** [comment]
> Hey @Sachin-chaurasiya , Can you assign me? I can work on it.

Assigned to you @jayanth137.

**Comment by priyankarpal** [comment]
Okay so @jayanth137 currently I'm unassigning you it's been a few months since it opened. if anyone wants to work please comment

**Comment by nirutthyu** [comment]
Hey @priyankarpal,can I work on this issue?

**Comment by nirutthyu** [comment]
Hey @Sachin-chaurasiya,can i work on this?

**Comment by Sachin-chaurasiya** [comment]
> Hey there,can i work on this?

Hey @nirutthyu , I have assigned the issue to you.
